app/admin/news_article.rb

```diff
@@ -0,0 +1,51 @@
+if defined?(ActiveAdmin)
+  ActiveAdmin.register News::Article, as: 'News' do
+    index do
+      selectable_column
+      column :id
+      column :title
+      column :annonce
+      actions
+    end
+
+    show do
+      attributes_table do
+        row :id
+        row :title
+        row (:text)  { |article| raw(article.text) }
+        row :text
+      end
+    end
+
+    form do |f|
+      inputs do
+        input :title
+        input :annonce
+        input :text, :input_html => { class: 'ckeditor' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.